consoles/sshVirtsh.pm

```diff
@@ -76,7 +76,7 @@ sub activate {
         die "cant' start xterm on $display (err: $! retval: $?)";
     }
     # FIXME: assert_screen('xterm_password');
-    sleep 3;
+    sleep 10;
```

Comment:
Isn't there anything we can assert before typing the password? E.g. output on a simulated serial console?